.github/workflows/test.yml

```diff
@@ -6,7 +6,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Probably should leave this change out here, and do it in #196.

Reply:
Removed, thanks!